tran_kien_giang/extend.php

```diff
@@ -0,0 +1,46 @@
+<?php 
+	$so_tien_can_quy_doi=$_GET['so_tien_can_quy_doi'];
+	$option=$_GET['option'];
```

Comment:
```suggestion
	$option = $_GET['option'];
```
trước và sau `=` có 1 dấu cách